packages/example/src/pages/index.tsx

```diff
@@ -4,10 +4,12 @@ import {
   withDefault,
   BooleanParam
 } from 'next-query-params';
+import { useRouter } from 'next/dist/client/router';
```

Comment:
```suggestion
import { useRouter } from 'next/router';
```

Also please run ESLint with `--fix` in the example.